src/components/installments/InstallmentPaymentStateInfo.tsx

```diff
@@ -0,0 +1,143 @@
+import classNames from 'classnames';
+import React, { useCallback } from 'react';
+import {
+  Button,
+  Dialog,
+  IconErrorFill,
+  IconAlertCircleFill,
+  IconCheckCircleFill,
+  IconInfoCircle,
+  Table,
+} from 'hds-react';
+import { useTranslation } from 'react-i18next';
+
+import formattedCurrency from '../../utils/formatCurrency';
+import { ApartmentInstallmentPayment, ApartmentInstallmentPaymentState } from '../../types';
+import formatDateTime from '../../utils/formatDateTime';
+
+import styles from './InstallmentPaymentStateInfo.module.scss';
+
+const T_PATH = 'components.installments.InstallmentPaymentStateInfo';
+
+interface IProps {
+  state: ApartmentInstallmentPaymentState;
+  payments: ApartmentInstallmentPayment[];
+  sentToSap: boolean;
+}
+
+type StatusLevel = 'unimportant' | 'success' | 'important' | 'critical';
```

Comment:
I would define these as either enums or const, as they are used in a few places in this file. As a const:

```
const StatusLevel = {
  UNIMPORTANT: "unimportant",
  SUCCESS: "success",
  IMPORTANT: "important",
  CRITICAL: "critical",
} as const;
```
Then the type would be:
```
type StatusLevelValue = typeof StatusLevel[keyof typeof StatusLevel];
```
And usage:
```
StatusLevel.CRITICAL
```